### PR TITLE
Add test tag with Oracle

### PR DESCRIPTION
**Pull request by sayden**
Adds the TEST_TAGS environment variable to Jenkins CI set to `aws,azure,oracle`

Continues / Related to: https://github.com/elastic/beats/pull/16937
Related to: https://github.com/elastic/beats/pull/17075

Pinging @jsoriano and @kuisathaverat

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -574,6 +574,7 @@ def withBeatsEnv(Closure body){
     "TEST_COVERAGE=true",
     "RACE_DETECTOR=true",
     "PYTHON_ENV=${WORKSPACE}/python-env",
+    "TEST_TAGS"="aws,azure,oracle",
```
Comment (jsoriano) [inline Jenkinsfile]: Please, don't add aws and azure yet, we have to add credentials and automate the setup of environments.

Comment (kaiyan-sheng) [reply Jenkinsfile]: I'm interested in the CI error with aws/azure in TEST_TAGS 😄

Comment (sayden) [reply Jenkinsfile]: I'll wait until it finishes before updating the branch :smile:

Comment (jsoriano) [reply Jenkinsfile]: Running aws tests locally I have found that I broke them :man_facepalming:  Preparing fix...

Comment (jsoriano) [reply Jenkinsfile]: Fix here: https://github.com/elastic/beats/pull/17084

When this is fixed, tests fail like this when `TEST_TAGS` includes `aws` and no credentials are provided:
```
--- FAIL: TestFetch (0.00s)
    cloudwatch_integration_test.go:20: $AWS_ACCESS_KEY_ID not set or set to empty
--- FAIL: TestData (0.00s)
    cloudwatch_integration_test.go:33: $AWS_ACCESS_KEY_ID not set or set to empty
FAIL
FAIL	github.com/elastic/beats/v7/x-pack/metricbeat/module/aws/cloudwatch	0.031s
```

File: Jenkinsfile [hunk 2]
```diff
@@ -574,6 +574,7 @@ def withBeatsEnv(Closure body){
     "TEST_COVERAGE=true",
     "RACE_DETECTOR=true",
     "PYTHON_ENV=${WORKSPACE}/python-env",
+    "TEST_TAGS"="oracle",
```
Comment (kuisathaverat) [inline Jenkinsfile]:
```suggestion
    "TEST_TAGS=oracle",
```

Comment (sayden) [reply Jenkinsfile]: Ohhh good catch :smile:

**Comment by kaiyan-sheng** [comment]
Could you change the PR title please to remove aws and azure 😬

**Comment by sayden** [comment]
> LGTM, but would you like to move this change to #16833 so both things are tested and merged together?

Ok, let's close this and see everything working together

**Pull request closed without merging**